### PR TITLE
adapt output precision in ScaLAPACK test 5

### PR DESCRIPTION
**Pull request by BenBrands**

### CODE REVIEW

File: tests/scalapack/scalapack_05.cc [hunk 1]
```diff
@@ -62,7 +62,7 @@ void test(const unsigned int size, const unsigned int block_size)
   const NumberType rcond = scalapack_A.reciprocal_condition_number(scalapack_l1);
 
 
-  pcout << 1./(l1*inv_l1) << " " << rcond << std::endl;
+  pcout << 1./(l1*inv_l1) << " " << std::setprecision(7) << rcond << std::endl;
```
Comment (davydden) [inline tests/scalapack/scalapack_05.cc]: from the output i noticed that `std::setprecision(7)` is being applied to both numbers from the second run. To make this consistent, could you please make it

```
pcout << std::setprecision(7) << 1./(l1*inv_l1) << " " << rcond << std::endl;
```

Comment (davydden) [reply tests/scalapack/scalapack_05.cc]: (and adjust the output files)

**Comment by davydden** [comment]
should fix https://cdash.kyomu.43-1.org/testSummary.php?project=1&name=scalapack%2Fscalapack_05.mpirun%3D1.debug&date=2018-01-24

File: tests/scalapack/scalapack_05.cc (hunk 1 above)
Comment (masterleinad) [reply tests/scalapack/scalapack_05.cc]: In fact, `pcout << std::setprecision(7)` modifies `pcout` not just for this statement, but for all following. Hence, I would just put it after the initialization of the `Utilities::MPI::MPI_InitFinalize` object.

Comment (davydden) [reply tests/scalapack/scalapack_05.cc]: yes, that's even better.

File: tests/scalapack/scalapack_05.mpirun=1.output [hunk 2]
```diff
@@ -1,36 +1,36 @@
 32 32 1 1
-0.410521 0.471758
```
Comment (davydden) [inline tests/scalapack/scalapack_05.mpirun=1.output]: does this output pass the test? I would expect the first number to have more digits

**Comment by davydden** [comment]
./run-tests

**Pull request merged**